server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/remote/delivery/DeliveryRetryHelperTest.java

```diff
@@ -66,15 +68,19 @@ public void incrementRetriesShouldWorkOnNonInitializedMails() throws Exception {
 
     @Test
     public void retrieveRetriesShouldBeZeroOnInvalidValue() throws Exception {
-        FakeMail mail = FakeMail.builder().attribute(DeliveryRetriesHelper.DELIVERY_RETRY_COUNT, "invalid").build();
+        FakeMail mail = FakeMail.builder()
+            .attribute(new Attribute(DeliveryRetriesHelper.DELIVERY_RETRY_COUNT, AttributeValue.of("invalid")))
```

Comment:
Can we have a `attribute(AttributeName, AttributeValue)` to reduce a bit the boiler plate in tests?

Or even a `attribute(AttributeName, Object value)` to reduce greatly this diff size.